omnibus/files/private-chef-cookbooks/private-chef/recipes/nginx.rb

```diff
@@ -56,14 +56,21 @@
    ssl_keyfile = File.join(nginx_ca_dir, "#{node['private_chef']['nginx']['server_name']}.key")
    ssl_crtfile = File.join(nginx_ca_dir, "#{node['private_chef']['nginx']['server_name']}.crt")
 
+   server_name = node['private_chef']['nginx']['server_name']
+   server_name_type = if server_name =~ OmnibusHelper.is_ip?(server_name)
```

Comment:
*jedi hand wave* There was no rookie mistake here, what are you talking about?

Good catch, forgot to commit some local change.  Now with correct code and tests!